config/services/node-exporter.service

```diff
@@ -0,0 +1,51 @@
+[Unit]
+Description=Prometheus Node Exporter
+Requires=network-online.target nginx.service
+After=network-online.target nginx.service
+
+[Service]
+StandardOutput=journal+console
+StandardError=journal+console
+Type=simple
+
+ExecStart=/bin/node_exporter \
+    --web.listen-address :9000 \
+    --web.telemetry-path /metrics \
+    --no-collector.arp \
```

Comment:
since you're disabling all collectors except one, can you explicitly enable that as well to make this easier to understand?